Dockerbuild fails

**Describe the bug**
The node version is quite old and it seems to break in the npm intsall phase
```
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @esbuild/android-arm@0.15.12: wanted {"os":"android","arch":"arm"} (current: {"os":"linux","arch":"x64"})

file:///usr/src/dispatch/src/dispatch/static/dispatch/node_modules/vite/bin/vite.js:7
    await import('source-map-support').then((r) => r.default.install())
    ^^^^^

SyntaxError: Unexpected reserved word
    at Loader.moduleStrategy (internal/modules/esm/translators.js:140:18)
    at async link (internal/modules/esm/module_job.js:42:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dispatch@0.1.0 build: `vite build --out-dir dist`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dispatch@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

````

I DID clear my build cache etc to clean out node_modules etc

**To Reproduce**
docker build -t dispatch:latest  .

**Expected behavior**
This should work

**Screenshots**
If applicable, add screenshots to help explain your problem.



**Additional context**
Add any other context about the problem here.
Mostly the forums point to a old version of nodejs

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -16,7 +16,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 RUN set -x \
-    && wget --quiet -O - https://deb.nodesource.com/setup_12.x | bash - \
+    && wget --quiet -O - https://deb.nodesource.com/setup_16.x | bash - \
     && apt-get install -y nodejs
 
 ARG SOURCE_COMMIT

